### PR TITLE
Logs similar to kube-burner.

### PR DESCRIPTION
**Pull request by smanda99**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [X] Optimization
- [ ] Documentation Update

## Description

Logs similar to kube-burner

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Comment by rsevilla87** [comment]
Hey @smanda99!, thanks for the PR, have you checked it is working correctly?

**Comment by smanda99** [comment]
Yes @rsevilla87, I have checked it. it collects logs like this :
image

File: cmd/ocp.go [hunk 1]
```diff
@@ -66,6 +66,7 @@ func openShiftCmd() *cobra.Command {
 			return
 		}
 		util.ConfigureLogging(cmd)
+		util.SetupLogging("ocp-"+workloadConfig.UUID)
```
Comment (rsevilla87) [inline cmd/ocp.go:69]: You need to add surrounding spaces to "+" to pass linting rules

```suggestion
		util.SetupLogging("ocp-" + workloadConfig.UUID)
```

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**